src/bone.js

```diff
@@ -1016,6 +1016,9 @@ class Bone {
       if (columnInfo && attribute.type instanceof DataTypes.DATE && attribute.type.precision == null) {
         attribute.type.precision = columnInfo.datetimePrecision;
       }
+      // if (columnInfo && columnInfo.defaultValue != null && attribute.defaultValue === undefined) {
+      //   attribute.defaultValue = columnInfo.defaultValue;
+      // }
```

Comment:
loading `defaultValue` from database, not activated yet due to not tested thoroughly, pending on @JimmyDaddy

Reply:
hmm....I'll check it